api/app/outcome/repository.py

```diff
@@ -28,5 +28,19 @@ def get_latest_for_event(event_id):
     @staticmethod
     def add(outcome):
         db.session.add(outcome)
+    
```

Comment:
It's probably worth removing the get_*_for_event methods now that outcome is linked to a response? That way we can also ensure all the relevant code paths are updated.